package.json

```
@@ -7,13 +7,15 @@
     "build": "next build",
     "start": "next start",
     "deploy": "next build && sls deploy",
+    "undeploy": "sls undeploy",
```

I'm not sure when this line was introduced, but I don't think `sls undeploy` will work. It needs to be `sls remove`.
```suggestion
    "undeploy": "sls remove",
```

---

Oh I forgot about this! I've tried to use `npm run undeploy` a few times, which exists in the Courses API, so quickly added it here without thinking it through properly.